util/tidy.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2013 The Servo Project Developers.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
```

Comment:
I definitely am not interested in dual licensing with Apache 2.0. It would be better to just paste in the MIT license here, instead of this statement that references those two licenses.